.github/CODEOWNERS

```diff
@@ -42,13 +42,6 @@
 /model/registry/url.yaml   @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-http-approvers
 /model/registry/user-agent.yaml   @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-http-approvers
 /docs/http/   @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-http-approvers
-/docs/attribute-registry/http.md   @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-http-approvers
```

Comment:
Why were these removed? 🤔

Reply:
There's a discussion about this above.  TL;DR; we don't want users touching the *markdown* of the reigstry, just the YAML.